ui-manchette/src/hooks/isOverFlowed.ts

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useLayoutEffect, useRef, RefObject } from 'react';
+
+interface CallbackFunction {
+  (hasOverflow: boolean): void;
+}
+
+interface CallbackFunction {
+  (hasOverflow: boolean): void;
+}
```

Comment:
As in osrd, use type instead of interface and delete duplications
```suggestion
type CallbackFunction = (hasOverflow: boolean) => void;
```

Reply:
done